000-vip-init.php

```diff
@@ -303,6 +304,11 @@
 
 	$encloseme_cleaner = new VIP_Encloseme_Cleanup();
 	$encloseme_cleaner->init();
+
+	// redirects url for jetpack network launches, if needed
+	require_once __DIR__ . '/config/class-vip-jetpack-network-launch-redirects.php';
+
+	add_action( 'init', [ VIP_Jetpack_Network_Launch_Redirects::class, 'maybe_redirect_jetpack_network_launches' ] );
```

Comment:
For safety during the initial rollout, will need to have this look more like:

```
if ( file_exists( __DIR__ . '/config/class-vip-jetpack-network-launch-redirects.php' ) ) {
	require_once __DIR__ . '/config/class-vip-jetpack-network-launch-redirects.php';
	add_action( 'init', [ VIP_Jetpack_Network_Launch_Redirects::class, 'maybe_redirect_jetpack_network_launches' ] );
}
```

Can remove the file_exists() on a subsequent deploy.

Would also prefer to move this functionality into more JP-specific area of MU. Let's put it into `vip-jetpack`.